src/db/ClimbSchema.ts

```diff
@@ -59,6 +59,28 @@ const MetadataSchema = new Schema<IClimbMetadata>({
   }
 }, { _id: false })
 
+const PitchSchema = new mongoose.Schema({
+  _id: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    default: () => muuid.v4()
+  },
+  uuid: {
+    type: 'string',
+    default: function () { return this._id.toString() }
+  },
+  parentId: { type: String, required: true },
+  number: { type: Number, required: true },
```

Comment:
`number` is to track the sequence of pitches. It is also only required field besides `id` and `parentId`. The reasoning: the correct sequence of information is essential for multi-pitches. A user might only add the pitches' `grades`; but it is prime that she does so in correct order, else the value of information is almost zero.

Since this seems to be non-intuitive, should I rename this field for more clarity? E. g. `pitchNumber`? (Suggestions welcome)

Reply:
We should use something else besides `number` to avoid conflict with `number` in Typescript. `pitchNumber` sounds good.  At first I thought pitch number isn't needed since we can get them from the array index.  But in some case it may not be easy to get the index.  Glad you thought of it.